### ISSUE TITLE
[Screen reader - Search]: Screen reader does not announce the 'Search Results' under the 'Find in document'

### ISSUE THREAD

**Issue by seahawksq** [mozilla/pdf.js#14525]
Screen reader user won't get search suggestions/ results if that suggestion is not being announced by screen reader and also user will have to navigate manually which will consume user's time & effort. Not only this, he/she will not get an idea that input given by him/her is validating the correct data or not. 
![image](https://user-images.githubusercontent.com/25859586/152266511-59dccde7-b750-44d6-b3d5-f246d3220ffe.png)

Configuration:
- Web browser and its version: Chrome Version 97.0.4692.71 (Official Build) (64-bit) /Edge Version 97.0.1072.69 (Official build) (64-bit)
- Operating system and its version: 21H2(OS build 22000.376)
- PDF.js version: 2.13.107
- Is a browser extension: No

Steps to reproduce the problem:
1. Navigate to https://mozilla.github.io/pdf.js/web/viewer.html
2. Open 'Find in document' edit field',
3. Reach out "Search text box" and enter any valid input.

What is the expected behavior? (add screenshot)
Screen reader should announce search results found on searching something valid in edit field i.e., "1 of 18 results Found" or the "Phrase not found" message on searching something invalid.

Screen Reader Tested:
 Chromium+Narrator and Chrome+NVDA.

**Comment by Snuffleupagus** [comment]
Based on skimming through https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/status_role and https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_status_role, would adding something like `role="status"` on the [relevant viewer DOM elements](https://github.com/mozilla/pdf.js/blob/3635a9a3338feb0049be4f1521d5a5afd173a832/web/viewer.html#L144-L149) be sufficient in order to fix this issue?

**Comment by JLavalleyENet** [comment]
The new release (2.14.305) does *not* appear to resolve this issue.  The work-item associated with this specifically references Firefox, but the report, as documented is for Chrome.

Here is an example of the problem, in Edge, as of 5/16/2022 2pm (eastern):
![image](https://user-images.githubusercontent.com/50142510/168654960-1b8e4da3-69d7-4f85-bd11-efd6f3edec2c.png)
